utils/guidelines_xslt/odd2html/guidelines.xsl

```diff
@@ -400,11 +400,56 @@
     
     <xd:doc>
         <xd:desc>
-            <xd:p>term</xd:p>
+            <xd:p>Names</xd:p>
+        </xd:desc>
+    </xd:doc>
+    <xsl:template match="tei:name" mode="guidelines">
+        <xsl:choose>
+            <xsl:when test="not(@ref)">
+                <xsl:apply-templates select="node()" mode="#current" />
+            </xsl:when>
+            <xsl:otherwise>
+                <a class="link_ref link_external" href="{@ref}">
+                    <xsl:apply-templates select="node()" mode="#current" />
+                </a>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
```

Comment:
This will affect the names of editors of the docs only. That content is contained in the headers of our guideline chapter files, which is not directly rendered to the web version of the Guidelines, and also not included in the PDF version. Instead, a special page is generated, which also deduplicates that content. In essence, this code has currently no effect.